js/mexc.js

```diff
@@ -623,13 +624,7 @@ module.exports = class mexc extends Exchange {
 
     async fetchTickers (symbols = undefined, params = {}) {
         await this.loadMarkets ();
-        const defaultType = this.safeString2 (this.options, 'fetchTickers', 'defaultType', 'spot');
-        const type = this.safeString (params, 'type', defaultType);
-        const query = this.omit (params, 'type');
-        if (type !== 'swap') {
-            throw new NotSupported (this.id + ' fetchTickers() is supported for swap markets only');
-        }
-        const response = await this.contractPublicGetTicker (query);
+        const response = await this.contractPublicGetTicker (params);
```

Comment:
This isn't really fixing the issue. Instead it will only work for contract markets, and will not work for spot markets.
In general, the exchanges might have different types of markets, which are not meant to be loaded at once.
Therefore we have to load them separately.

This specific exchange has two types of markets: spot and swap.

To instantiate it with spot-markets:

```Python
# SPOT usage
exchange = ccxt.mexc({
    'options': {'defaultType': 'spot'}
})
tickers = exchange.fetch_tickers()  # will throw an exception in the master
```

To instantiate it with swap-markets:

```Python
# SWAP usage
exchange = ccxt.mexc({
    'options': {'defaultType': 'swap'}
})
tickers = exchange.fetch_tickers()  # will work
```

The library works for swaps, but throws an exception on spot markets.
We should fix that by changing the code of the spot-case, so that it works and does not throw.
In order to do that, we must differentiate between the two endpoints, depending on the supplied defaultType.

Currently, `fetchTickers` only works for swaps. The user has reported that this exchange has a fetchTickers capability, if you omit the symbol. So we can make `fetchTickers` work for `spot` markets as well.

See how that is done in this example:

- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1933
- https://github.com/ccxt/ccxt/blob/master/js/mexc.js#L670

In this PR, you're only calling the contract (swap) endpoint. We should replace the code as follows:

```JavaScript
        if (type !== 'swap') {
            throw new NotSupported (this.id + ' fetchTickers() is supported for swap markets only');
        }
        const response = await this.contractPublicGetTicker (this.extend (request, params));
```
↓
```JavaScript
        let method = 'spotPublicGetMarketTicker';
        if (type === 'swap') {
            method = 'contractPublicGetTicker';
        }
        const response = await this[method] (this.extend (request, params));
```

After doing the above, we must also make sure that the response-parsing stage of fetchTickers works well for both types of tickers, spot and swap.